package.json

```diff
@@ -7,8 +7,13 @@
     "@shopify/polaris-icons": "^4.5.0",
     "date-fns": "^2.21.1",
     "date-fns-tz": "^1.1.4",
+    "i18next": "^20.2.2",
+    "i18next-browser-languagedetector": "^6.1.0",
+    "i18next-fs-backend": "^1.1.1",
```

Comment:
Don't think we need these two packages... they are for retrieving translations from a backend, which we are not doing. We are bundling our translations.

Reply:
i18next-browser-languagedetector detects the current language based off of a browser's language settings. Try changing the preferred language to different languages ("Choose your preferred language for displaying pages" in Firefox) and see what happens (use incognito mode). It should display French if language is set to fr-CA, fr-FR, etc., English if language is set to en-CA, en-US, etc. and English if any other language is set.

i18next-fs-backend is used in the tests to load the translation.json files, it is not used anywhere else.

Reply:
I meant `i18next-fs-backend` and `i18next-http-backend` both of which are used only to pull in our translations via AJAX instead of just adding them directly to our bundle. I don't see the need to pull in our translations via AJAX... It means we need to add two deps, our configuration is more complicated, and loading the translations is a bit slower as it means initiating an ajax request after our app is loaded.